### PR TITLE
Using constants for Rinkeby tokens

### PR DESCRIPTION
**Pull request by dmosites**
Also includes an error syntax update for the JSON-RPC client.

### CODE REVIEW

File: maker.test.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const ethers = require('ethers')
```
Comment (alicevhenshaw) [inline maker.test.js]: is it worth importing ethers purely to get the zero address from it?

File: scripts/constants.js [hunk 2]
```diff
@@ -3,8 +3,11 @@ module.exports = {
   MAX_LOCATORS: 10,
   APPROVAL_AMOUNT: '90071992547409910000000000',
   INDEX_HEAD: '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF',
-  NULL_ADDRESS: '0x0000000000000000000000000000000000000000',
   DEFAULT_CONFIRMATIONS: 2,
+  rinkebyTokens: {
+    DAI: '0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea',
```
Comment (poiresel) [inline scripts/constants.js]: do these need to be checksummed?

Comment (dmosites) [reply scripts/constants.js]: They do not need to be checksummed. Let me know if you have something in mind

Comment (poiresel) [reply scripts/constants.js]: INDEX_HEAD: '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF' is checksummed which was why i asked.

Comment (poiresel) [reply scripts/constants.js]: if not required, all good!

**Pull request merged**